src/views/ConsentContinuationOut.vue

```diff
@@ -65,7 +65,7 @@
             <section>
               <header>
                 <h2>Ledger Detail</h2>
-                <p>Enter a detail that will appear on the ledger for this entity.</p>
+                <p class="grey-text">Enter a detail that will appear on the ledger for this entity.</p>
```

Comment:
`<jedi>`I don't think this is the colour you're looking for.`</jedi>`

The UX team specified a colour of `#495057`, which we declare in theme.scss as "$gray7". You can use this variable in CSS, but in the template (as here), you can use a named colour (eg, magenta) or a Vuetify theme colour -- declared [here](https://github.com/bcgov/business-filings-ui/blob/b7c7f60caec253bb45a78ae0d6fd8765ad5eba35/src/main.ts#L100).

Your choices are:
- add gray7 to the theme (and ideally update all the code that does it the other way), or
- use a class that specifies the needed colour

Reply:
Search the project for "$gray7".

If you're lucky, there will already be a base class (in base.scss) that specifies what you need (assuming you haven't overridden it somehow).

Reply:
Wait Sev, I'm confused. In the CSS, I have grey-text set the color as $gray7. Line: (https://github.com/JazzarKarim/business-filings-ui/blob/eb63b323e2786770409f6501de3b12d008da02f2/src/views/ConsentContinuationOut.vue#L959)

Isn't this what you mean?

Reply:
Oh, my fault, I didn't see this.

I skipped to a conclusion -- it's possible to specify a `color` in some Vuetify components, and those follow the theme as above. But this isn't what you're doing 😛 

Yes, you're doing the right thing.